programs/bpf_loader/src/lib.rs

```diff
@@ -9,6 +9,7 @@ use libc::c_char;
 use log::*;
 use solana_rbpf::{EbpfVmRaw, MemoryRegion};
 use solana_sdk::account::KeyedAccount;
+use solana_sdk::credit_only_account::KeyedCreditOnlyAccount;
```

Comment:
Side note, from a dev perspective the long name is annoying, they see accounts with its corresponding keys as they would expect.  We only call them KeyedAccounts because internally we use the name account to refer to just accounts without keys.  Would be nice to invert that.

Reply:
Maybe just call them `RWAccount` and `ROAccount`?

Reply:
Completely agree